fedimint-core/src/db/mod.rs

```diff
@@ -1823,6 +1823,8 @@ pub async fn apply_migrations(
 ) -> Result<(), anyhow::Error> {
     let mut dbtx = db.begin_transaction().await;
     let disk_version = dbtx.get_value(&DatabaseVersionKey).await;
+
+    // FIXME: should this be a loop if multiple migrations have to happen?
```

Comment:
there is already a loop 

https://github.com/fedimint/fedimint/blob/9c36777c4a0f21742bc99c29c54c12d3f74a4e52/fedimint-core/src/db/mod.rs#L1835

Reply:
I'm just blind, never mind 🤦‍♂️